devops/scripts/benchmarks/main.py

```diff
@@ -310,13 +345,13 @@ def main(directory, additional_env_vars, save_name, compare_names, filter):
             f"Markdown with benchmark results has been written to {md_path}/benchmark_results.md"
         )
 
-    saved_name = save_name if save_name is not None else this_name
+    saved_name = options.save_name if options.save_name is not None else this_name
 
     # It's important we don't save the current results into history before
     # we calculate historical averages or get latest results for compare.
     # Otherwise we might be comparing the results to themselves.
     if not options.dry_run:
-        history.save(saved_name, results, save_name is not None)
+        history.save(saved_name, timestamp, results, options.save_name is not None)
```

Comment:
Now `history.save` can use `options.save_name` directly and the last argument can be removed